Gemfile

```diff
@@ -44,5 +47,9 @@ group :development do
   gem "spring-watcher-listen", "~> 2.0.0"
 end
 
+group :test do
+  gem 'simplecov', require: false
+end
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem "tzinfo-data", platforms: %i[mingw mswin x64_mingw jruby]
+gem 'jwt'
```

Comment:
Remove this, jwt token will be in separate PR

Reply:
done